Xamarin.Forms.Platform.UAP/ScrollViewRenderer.cs

```diff
@@ -4,6 +4,7 @@
 using Windows.Foundation;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using UwpScrollBarVisibility = Windows.UI.Xaml.Controls.ScrollBarVisibility;
```

Comment:
Needed to alias one of ScrollBarVisibility enums, decided to do that to the platform enum instead of forms